worlds/dlcquest/Options.py

```diff
@@ -94,6 +94,15 @@ class ItemShuffle(Choice):
     default = 0
 
 
+dlc_quest_option_classes = [
+    DoubleJumpGlitch,
+    TimeIsMoney,
+    CoinSanity,
+    CoinSanityRange,
+    EndingChoice,
+    Campaign,
+    ItemShuffle,
+]
```

Comment:
```suggestion
]

```

Reply:
Resolved after the rebase from post-993 main